docs/drafts/BEP-0001.md

```diff
@@ -41,14 +41,17 @@ Adding a populationAbundance() function would allow capturing population level c
  
 ## Discussion
 
-* Should we use this for cell populations as well?
+* Should we use this for cell populations as well? E.g., "T-helper 17 cell"?
```

Comment:
I would like to see some examples of what types of knowledge this would be used to represent, and where this might be useful in reasoning/analysis of networks. 

Would this also mean introducing another new function like `cell()`?

Reply:
populationAbundance(CELL:"Dendritic Cells") should capture cell population abundances correctly.  

An example is a(DRUG:X) decreases pop(CELL:"Dendritic Cells").

Reply:
Will update with an example. I think we need a means of representing changes in cell populations. The question here is if pop() can be used for both cell populations and populations of taxonomic groups, or if another entity type cell() - and another BEP - is needed.